### ISSUE TITLE
Collector - Downloading Map for Offline Use - Only with Online Basemap and not tile packages?

### ISSUE THREAD

**Issue by atjalloh** [doug-m/CollectorAuroraTesting#107]
### Issue
<Side load tile package for offline use -->

### Expected Behavior

* Group Name:
* Map Name:

### Device Information
* Device: I phone 
* OS Version: 11.4

**Comment by FsenyahPHE** [comment]
Yes, this is a key requirement for our workflow also.  What is the workflow (if available) to take a sideloaded basemap offline using AGOL?  I can sideload and access a raster basemap to work with while online but I can’t select a basemap to take offline like current Collector.  I would expect the following behaviour:

'Add Offline Area' - Access sideloaded basemaps - Option to select sideloaded basemap as offline area aswell as current AGOL basemap

Device: Iphone 7

**Comment by amc112** [comment]
Same here. Being able to sideload TPK files to Collector is a necessary step in our implementation.

Here's a screenshot describing how we currently copy TPKs to Collector on iOS... [https://collector-holistic.slack.com/messages/DB8V3J6LQ/convo/C9Z5JDNUQ-1529356565.000464/](url). This didn't work for Collector Aurora.

**Comment by atjalloh** [comment]
Hi.
I just want to give some context to what amc11 and I have just noted with this new Aurora Beta release "offline" component not having the option to side your own "datasets" via tile packages.

The concept of Tile packages and the ability to side load them into our devices is what has made Collector viable and attractive to us as a Mineral Exploration entity and I am sure to many others.

We use various types of datasets in the field not just using orthorectified imagery as a basemap. 

We work with  Geochemical, Geophysical and other types of datasets that the field worker needs to switch back on forth in Collector when they are out in the field to aid in their interpretation.

It is an "iterative" process.

The GIS adminsitrator, for a truly "offline" scenario (remote areas with little or no wifi connectivity) really has only one opportunity to download all these datasets that are required for that field worker to use in the field i.e. before they go to that area and use Collector with the workflow described above.

Lastly, tile packages do not limit the amount of data that can be loaded into the device as is the case with downloading data from Portal for ArcGIS or ArcGIS online.

I say this again, the concept of tile packages, made Collector more justifiable to use in a truly "offline" environment.

We look forward to seeing this being reinstated in the next release based on the assurances received yesterday.

Thank you.

Regards,
Aziz
